tools/vault/config/setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+export VAULT_ADDR='https://vault.apps.digilab.network'
```

Comment:
Is it needed to hardcode this address? It makes it less portable.

Reply:
this is where the service is running. if you want to change it you need to also change the ingress (which is hardcoded). would you prefer me nog hardcoding is and making a -z check?

Reply:
For now I think it is fine to keep it this way, changing it may be more effort than the gain at this point. Just noticed it and wanted to verify this was by choice.